package.json

```diff
@@ -2,14 +2,23 @@
   "name": "sk-auth",
   "version": "0.3.7",
   "description": "Authentication library for use with SvelteKit featuring built-in OAuth providers and zero restriction customization!",
-  "main": "dist/index.js",
-  "module": "dist/index.esm.js",
-  "types": "dist/index.d.ts",
+  "module": "index.mjs",
```

Comment:
I'm sure I'm not alone when I say `.mjs` files were one of the silliest decisions the Node community made. What they should've done was used `.njs` for anything which required a Node runtime to function.